src/med_bench/mediation.py

```diff
@@ -6,24 +6,29 @@
 # first step, run r code to have the original implementation by Huber
 # using rpy2 to have the same data in R and python...
 
+from itertools import combinations
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
 import rpy2.robjects as robjects
 import rpy2.robjects.packages as rpackages
-from rpy2.robjects import pandas2ri, numpy2ri
-from sklearn.linear_model import LogisticRegressionCV, RidgeCV, LassoCV
-from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
-from sklearn.preprocessing import PolynomialFeatures
-from sklearn.calibration import CalibratedClassifierCV
-import numpy as np
 from numpy.random import default_rng
+from rpy2.robjects import numpy2ri, pandas2ri
 from scipy import stats
-import pandas as pd
-from pathlib import Path
-from scipy.stats import bernoulli
 from scipy.special import expit
-
-from itertools import combinations
+from scipy.stats import bernoulli
+from sklearn.calibration import CalibratedClassifierCV
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.linear_model import LassoCV, LogisticRegressionCV, RidgeCV
 from sklearn.model_selection import KFold
-from .utils.utils import get_interactions, _convert_array_to_R
+from sklearn.preprocessing import PolynomialFeatures
+
+from .utils.classifiers import (_estimate_f_mu, _estimate_f_mu_cross_mu,
```

Comment:
it should not be .utils.classifiers, as it it not only classifiers. utils.nuisances ?

Reply:
This has been addressed in 3f1b34577643b37b886202ab5873e7b5a9054a40